activerecord/lib/active_record/connection_adapters/abstract/schema_definitions.rb

```diff
@@ -357,8 +357,8 @@ def timestamps(**options)
       #
       # See {connection.add_reference}[rdoc-ref:SchemaStatements#add_reference] for details of the options you can use.
       def references(*args, **options)
-        args.each do |col|
-          ReferenceDefinition.new(col, **options).add_to(self)
+        args.each do |ref_name|
```

Comment:
Actually `args` is an array of `ref_name`, not `col`. And `**options` create new hash.

```ruby
irb(main):001:0> def foo(o)
irb(main):002:1>   o.object_id
irb(main):003:1> end
=> :foo
irb(main):004:0> a = {}
=> {}
irb(main):005:0> a.object_id
=> 70096840684060
irb(main):006:0> foo(a)
=> 70096840684060
irb(main):007:0> foo(**a)
=> 70096844445860
```

`TableDefinition` and `Table` classes exists in the doc. I think it is better to define consistent method signature.
Should I separate a PR as fixing the doc?

http://edgeapi.rubyonrails.org/classes/ActiveRecord/ConnectionAdapters/Table.html#method-i-references
http://edgeapi.rubyonrails.org/classes/ActiveRecord/ConnectionAdapters/TableDefinition.html#method-i-references